multimedia/tvheadend/files/tvheadend.config

```diff
@@ -4,7 +4,7 @@
 #	 option config_path '/etc/tvheadend'
 
 config tvheadend server
-	option ipv6 '1'
+	option ipv6 '0'
```

Comment:
Even I read the commit description, I must ask. Why? There is not reason to disable IPv6, if you are not using it in your use case.